src/rpm_ostree/cli_finalize.rs

```diff
@@ -0,0 +1,22 @@
+//! Interface to `rpm-ostree finalize-deployment`.
+
+use super::Release;
+use failure::{bail, format_err, Fallible, ResultExt};
+
+/// Unlock and finalize the new deployment.
+pub fn finalize_deployment(release: Release) -> Fallible<Release> {
+    let cmd = std::process::Command::new("rpm-ostree")
+        .arg("finalize-deployment")
+        .arg(release.reference_id())
```

Comment:
Though here, it expects the checksum, not a `revision=$checksum`. Offhand, I'd just drop the `reference_id()` thing and do the `format!()` directly in `locked_upgrade()`. Really, that special syntax is available just for `deploy` and `rebase`.

Reply:
Good catch. I guess I was assuming that the input parse was uniform across verbs. I'll undo the helper as you suggested.